torch/linalg/__init__.py

```diff
@@ -213,6 +213,66 @@
             [-3.1113,  2.7381]], dtype=torch.float64)
 """)
 
+matrix_rank = _add_docstr(_linalg.linalg_matrix_rank, r"""
+matrix_rank(input, tol=None, hermitian=False) -> Tensor
+
+Computes the numerical rank of a matrix :attr:`input`, or of each matrix in a batched :attr:`input`.
+The matrix rank is computed as the number of singular values (or the absolute eigenvalues when :attr:`hermitian` is ``True``)
+above the specified :attr:`tol` threshold.
+
+If :attr:`tol` is not specified, :attr:`tol` is set to
+``S.max(dim=-1) * max(input.shape[-2:]) * eps`` where ``S`` is the singular values
+(or the absolute eigenvalues when :attr:`hermitian` is ``True``),
+and ``eps`` is the epsilon value for the datatype of :attr:`input`.
```

Comment:
Is there a place for the user to discover what the dtype-specific eps values are that can be linked here?

Reply:
Yes, I found https://pytorch.org/docs/stable/type_info.html. I'll link it here.